docs/examples/sentence_embedding/bert.md

```diff
@@ -0,0 +1,254 @@
+# Fine-tuning Sentence Pair Classification with BERT
+
+Pre-trained language
+representations have been shown to improve many downstream NLP tasks such as
+question-answering, and natural language inference. To apply pre-trained
+representations to these tasks, there are two strategies: 1) **feature-based**
+approach, which uses the pre-trained representations as additional features to
+the downstream task. 2) **fine-tuning** based approach, which trains the
+downstream tasks by fine-tuning pre-trained parameters. 
+
+While feature-based
+approaches such as ELMo [3] (introduced in the previous tutorial) are effective
+in improving many downstream tasks, they require task-specific architectures.
+Devlin, Jacob, et al proposed BERT [1] (Bidirectional Encoder Representations
+from Transformers), which **fine-tunes** deep bidirectional representations on a
+wide range of tasks with minimal task-specific parameters, and obtained state-
+of-the-art results.
+
+In this tutorial, we will focus on fine-tuning with the
+pre-trained BERT model to classify semantically equivalent sentence pairs.
+Specifically, we will:
+1. load the state-of-the-art pre-trained BERT model.
```

Comment:
I had the impression that there needs to be an empty line before the first list item to make the formatting correct. Maybe not necessary for md but you can double check the rendered results.

Reply:
line added